mozregression/download_manager.py

```diff
@@ -234,7 +234,7 @@ def __init__(self, destdir, session=requests, persist_limit=None):
     def get_dest(self, fname):
         return os.path.join(self.destdir, fname)
 
-    def cancel(self, cancel_if=None):
+    def cancel(self, cancel_if=None, wait=False):
```

Comment:
Yeah I think that might be a better idea. Put the wait in the commandline file with a comment as to why it's there (maybe even linking to this bug #)